src/aleph_client/commands/instance/__init__.py

```diff
@@ -695,17 +709,17 @@ async def list(
             echo(messages.json(indent=4))
         else:
             # Since we filtered on message type, we can safely cast as InstanceMessage.
-            messages = cast(List[InstanceMessage], messages)
+            messages = cast(builtins.list[InstanceMessage], messages)
```

Comment:
I am wondering if we should not update `filter_only_valid_messages` to use [generics](https://docs.python.org/3/library/typing.html#generics) :thinking:

Reply:
I'm not sure it's in the scope of this PR? We probably want to use `Sequence` instead of `list` for it as a type anyway